### ISSUE TITLE
Is there a way to change the text color of the iframe's body when applying the dark theme?

### ISSUE THREAD

**Issue by Jak3b0** [hipstersmoothie/storybook-dark-mode#131]
Hi,

Is there a way to change the text color of the iframe's body when applying the dark theme? I can see the `color` attribute of the iframe set to `white` but it is not changing the default text color of my stories.

I followed the getting started and changed the following:

```
export const parameters = {
  darkMode: {
    dark: { ...themes.dark, textColor: 'white' },
    current: 'dark',
    stylePreview: true,
  },
};
```

Thanks! :)

**Comment by Jak3b0** [comment]
For more context, I am using TailwindCSS and I added a style for the body element to set the text white when the dark mode is enabled. Since this was not working in Storybook, I was trying to find another way to set the text color via storybook-dark-mode but I realized that in order for Tailwind to apply the text color when in dark mode, the `dark` need to be set on a parent element (so in my case, this needs to be set on the `html` element).

So, I added the following code in the `preview.ts` file in my `.storybook` folder:

```
import addons from '@storybook/addons';

// get an instance to the communication channel for the manager and preview
const channel = addons.getChannel();

// switch body class for story along with interface theme
channel.on('DARK_MODE', (isDark) => {
  if (isDark) {
    // Set the `dark` class on the html element
    document.documentElement.classList.add('dark');
  } else {
    // Remove the `dark` class on the html element
    document.documentElement.classList.remove('dark');
  }
});
```

Is there a way to tell `storybook-dark-mode` to set the `dark` class on the html element instead of the body element? This would allow mode to remove this workaround :)

Thanks!

**Comment by dricholm** [comment]
Hello @Jak3b0, a PR was just merged a few days ago to fix this: #144. It was released in [1.0.7](https://github.com/hipstersmoothie/storybook-dark-mode/releases/tag/v1.0.7).
[Preview class target](https://github.com/hipstersmoothie/storybook-dark-mode#preview-class-target) was added to the README. With it you can set the target to the `html` element.
I needed this for the same reason as you with TailwindCSS.

**Comment by hipstersmoothie** [comment]
Thanks @dricholm. Seem like this one is close now, great work 🎉
